Add Cloud Hypervisor and OpenJ9 files (#221)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/cloud-hypervisor.md

---
name: Cloud Hypervisor
category: Compilers/Tools

> description: Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) written in Rust, designed to execute contemporary cloud workloads with the least amount of hardware emulation possible.

Author: Plag found

download_url: https://github.com/cloud-hypervisor/cloud-hypervisor/releases
works_on_arm: true
supported_minimum_version: 
    version_number: v18.0
    release_date: 2021/09/09

optional_info:
    homepage_url: https://www.cloudhypervisor.org/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content:
        official_docs: https://www.cloudhypervisor.org/docs/prologue/quick-start/
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum: https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v18.0
    release_notes__recommended_minimum:

>     other_info: Experimental support for arm64 began added in [v0.8.0](https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v0.8.0). Full functionality are not included in the release. Kindly refer [here](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/v0.8.0/docs/arm64.md).

Author: ```suggestion
    other_info: Experimental support for arm64 was first added in [v0.8.0](https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v0.8.0). Full functionality is not included in this release. Kindly refer [here](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/v0.8.0/docs/arm64.md).
```
